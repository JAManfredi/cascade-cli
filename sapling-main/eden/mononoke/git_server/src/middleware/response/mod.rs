/*
 * Copyright (c) Meta Platforms, Inc. and affiliates.
 *
 * This software may be used and distributed according to the terms of the
 * GNU General Public License version 2.
 */

pub mod content_type;
#[cfg(fbcode_build)]
mod facebook;
pub mod ods3;
#[cfg(not(fbcode_build))]
mod oss;
