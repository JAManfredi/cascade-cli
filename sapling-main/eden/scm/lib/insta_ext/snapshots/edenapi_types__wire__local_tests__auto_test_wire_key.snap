---
source: mod.rs
expression: "WireKey::arbitrary(&mut g)"

---
{"0":"jaiffe/nuat/fk/ejkleav/rc/ohh","1":[38,85,25,36,58,55,101,239,1,100,81,21,29,1,4,212,7,211,178,104]}
